### PR TITLE
ui: DB Console branding refresh

### LINKED ISSUE #56959: ui: DB Console branding refresh
Update logo:

* Site header
* Login page
* Favicon 

Update primary color Green (#37A806) to Purple (#6933FF) on buttons and other primary states
Note: Success states like alerts and badges should stay green

Resources

[DB Console Brand Updates](https://www.figma.com/file/cxE2QFATR24EStFTeHNdJc/obsrv_console-brand-update?node-id=1%3A10)

[Figma Design System](https://www.figma.com/file/B5AtEGdRbW3VxiBdg62TB8Xw/CRDB-Design-System?node-id=6398%3A39)

Since the DB Console uses Stylus for CSS, it may be difficult to use the design tokens from the design system here: https://github.com/cockroachdb/ui/pull/137

It is *not* a requirement to use the design tokens but if there's a straightforward way to import these then please do. Our assumption is that making the modifications directly in the DB Console is fine for now but we don't want to add Stylus support to the UI repository since we're trying to phase it out.

### PR DESCRIPTION
**Pull request by vladlos**
Updated logo, favicon, primary color on buttons changed to purple, new link colors.
Updated sign up popup design.

Resolves: #56959

Release note (ui): minor style changes to represent new branding pallete

Screenshot 2020-11-25 at 19 07 21
Screenshot 2020-11-25 at 19 07 14
Screenshot 2020-11-25 at 19 07 03

### CODE REVIEW

**Review by koorosh — CHANGES_REQUESTED** [review]
> Reviewed 9 of 9 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster)

*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8IGXMCbexoGa9JdZl:-MN8IGXMCbexoGa9JdZm:btztee0) (no related file):*
Font colors on Login screen seem to be wrong. 

Screen Shot 2020-11-27 at 11 33 17 AM
Screen Shot 2020-11-27 at 11 39 35 AM

---
*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8IOIFAJUcxJrfSK9e:-MN8IOIG2gFQkMntBiSS:b-i8jk6n) (no related file):*
@elkmaster , in case of dark theme, should we preserve white color for paths inside of icon?


---
*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8MoOb0AGPD86vrTjH:-MN8MoOcB995B5CHqmyS:b-rypu2k) (no related file):*
@elkmaster , what is expected color for hovered links? Currently hovered links change colors to default Antd color

on node details page:


on node overview page for logs link:


---
___
*[pkg/ui/src/components/button/button.module.styl, line 65 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8Sn_J8xEeWDvUGL_i:-MN8Sn_KB6EiEmaGs-mT:b-sbtnl) ([raw file](https://github.com/cockroachdb/cockroach/blob/a6862d91191f4df6f21582f0bca93cab43e69a9c/pkg/ui/src/components/button/button.module.styl#L65)):*
> ```Stylus
> 
>   &.crl-button--disabled
>     border solid 1px #d0bfff
> ```

Background color for disabled controls has to be #CBBCFC. Could you add this color to `colors.styl` as `primary-purple-2`?


___
*[pkg/ui/src/views/dashboard/emailSubscription.styl, line 25 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8VK8y9O6B0fHymH8z:-MN8VK8y9O6B0fHymH9-:b34mtz8) ([raw file](https://github.com/cockroachdb/cockroach/blob/a6862d91191f4df6f21582f0bca93cab43e69a9c/pkg/ui/src/views/dashboard/emailSubscription.styl#L25)):*
> ```Stylus
>   background-image url("../../../assets/dashboard/email_signup_bg.png")
>   background-repeat no-repeat
>   background-color #161039
> ```

@elkmaster , it might be better to fill entire background with
`background-repeat: round;` so you won't need to define custom color at all.

**Review by vladlos — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh)

*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8IGXMCbexoGa9JdZl:-MNTzfUD5pmHtx93B4E_:b-896fix) (no related file):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

Font colors on Login screen seem to be wrong. 

Screen Shot 2020-11-27 at 11 33 17 AM
Screen Shot 2020-11-27 at 11 39 35 AM

</blockquote></details>

Done.

---
*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8IOIFAJUcxJrfSK9e:-MNTzgLK9qxtBTrcVkXp:b-u3fzsi) (no related file):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

@elkmaster , in case of dark theme, should we preserve white color for paths inside of icon?


</blockquote></details>

I don't know, will ask Anne

---
*[a discussion](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8MoOb0AGPD86vrTjH:-MNTzw_YEMn5RE1FENC9:b-896fix) (no related file):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

@elkmaster , what is expected color for hovered links? Currently hovered links change colors to default Antd color

on node details page:


on node overview page for logs link:


</blockquote></details>

Done.

---
___
*[pkg/ui/src/components/button/button.module.styl, line 65 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8Sn_J8xEeWDvUGL_i:-MNTzzKv-LG0CtVEmuBs:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/a6862d91191f4df6f21582f0bca93cab43e69a9c/pkg/ui/src/components/button/button.module.styl#L65)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

Background color for disabled controls has to be #CBBCFC. Could you add this color to `colors.styl` as `primary-purple-2`?



</blockquote></details>

Done.
___
*[pkg/ui/src/views/dashboard/emailSubscription.styl, line 25 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MN8VK8y9O6B0fHymH8z:-MNU-01oCP3KD9vUiHK8:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/a6862d91191f4df6f21582f0bca93cab43e69a9c/pkg/ui/src/views/dashboard/emailSubscription.styl#L25)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

@elkmaster , it might be better to fill entire background with
`background-repeat: round;` so you won't need to define custom color at all.
</blockquote></details>

Done.

**Review by koorosh — CHANGES_REQUESTED** [review]
> Reviewed 5 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster and @koorosh)

___
*[pkg/ui/styl/base/typography.styl, line 38 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNYMnnOEwEeUxfOIGXD:-MNYMnnOEwEeUxfOIGXE:b-kycdhs) ([raw file](https://github.com/cockroachdb/cockroach/blob/9d09976fd11184454bb25405846e7e8f19eae935/pkg/ui/styl/base/typography.styl#L38)):*
> ```Stylus
>   font-family $font-family--monospace
> 
> a
> ```

Do we need to define this as a global style? I would prefer to set these styles for `<Anchor>` component and then reuse this component instead of `<a>` element across the project.
Anchor component is basically a wrapper on top of `<a/>` element and inherits all its props.
___
*[pkg/ui/styl/base/typography.styl, line 38 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNYO-JC0siwYQcJrCL6:-MNYO-JC0siwYQcJrCL7:bjdavoo) ([raw file](https://github.com/cockroachdb/cockroach/blob/9d09976fd11184454bb25405846e7e8f19eae935/pkg/ui/styl/base/typography.styl#L38)):*
> ```Stylus
>   font-family $font-family--monospace
> 
> a
> ```

Also we use `<Link />` component from `react-router-dom` package which defines it's own styles and even this global styles don't affect it (for instance statements link)

**Comment by elkmaster** [comment]
cc @koorosh 
Screenshot 2020-12-02 at 16 10 22

**Review by vladlos — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh)

___
*[pkg/ui/styl/base/typography.styl, line 38 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNYMnnOEwEeUxfOIGXD:-MNcTUPM3ha2FmzGqB6I:b-mmeznp) ([raw file](https://github.com/cockroachdb/cockroach/blob/9d09976fd11184454bb25405846e7e8f19eae935/pkg/ui/styl/base/typography.styl#L38)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

Do we need to define this as a global style? I would prefer to set these styles for <Anchor> component and then reuse this component instead of <a> element across the project.
Anchor component is basically a wrapper on top of <a/> element and inherits all its props.
</blockquote></details>

this was added for nodes overview table, but eventually found a way to do it component specific.
___
*[pkg/ui/styl/base/typography.styl, line 38 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNYO-JC0siwYQcJrCL6:-MNd31-D1wSXUlT2m6mR:b2jgnwj) ([raw file](https://github.com/cockroachdb/cockroach/blob/9d09976fd11184454bb25405846e7e8f19eae935/pkg/ui/styl/base/typography.styl#L38)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

Also we use <Link /> component from `react-router-dom` package which defines it's own styles and even this global styles don't affect it (for instance statements link)
</blockquote></details>

actually it is affect but has some more specific rules in place. addressed this issue in pr for admin-ui-components

https://github.com/cockroachdb/admin-ui-components/pull/44/files#diff-44744cacf152c84230fcb54fea87b019421cb68f60ec66a656e9388789d69ddbL70

**Review by nathanstilwell — APPROVED** [review]
> Reviewed 3 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh)

File: pkg/ui/src/components/core/colors.styl [hunk 1]
```diff
@@ -47,13 +47,17 @@ $colors--primary-green-3 = #37a806
 $colors--primary-green-4 = #237300
 $colors--primary-green-5 = #123d00
 
+$colors--primary-purple-2 = #CBBCFC
+$colors--primary-purple-3 = #6933FF
+
+$colors--info-4 = #0037a5
+
 // Functional colors
 $colors--functional-purple-1 = #cec0f2
```
Comment (nathanstilwell) [inline pkg/ui/src/components/core/colors.styl:56]: I see the "primary" color palette (purple) defined in Figma. Where do the "functional-purple" colors come from?

**Review by koorosh — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster and @koorosh)

___
*[pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl, line 49 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNwfOtCBU1RY6uga-5a:-MNwfOtCBU1RY6uga-5b:bfkr2r) ([raw file](https://github.com/cockroachdb/cockroach/blob/397aef9225765a2ad07f378ef79914ae3d36129e/pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl#L49)):*
> ```Stylus
>   text-decoration underline
> 
> a.nodes-table__link, .nodes-table__link a
> ```

This doesn't fill like a consistent solution, instead of adjusting class names, I would suggest to refactor places where `nodes-table__link` class is used. Particularly, define this class exactly on `<Link>` or `<a>` elements

For instance, one of code blocks can be changed from this:
```jsx
<div className="cell--show-on-hover nodes-table__link">
    <Link to={`/node/${record.nodeId}/logs`}>Logs</Link>
</div>
```

to this:
```jsx
<div className="cell--show-on-hover">
    <Link to={`/node/${record.nodeId}/logs`} className="nodes-table__link">Logs</Link>
</div>
```

File: pkg/ui/src/components/core/colors.styl (hunk 1 above)
Comment (elkmaster) [reply pkg/ui/src/components/core/colors.styl:56]: those are some old vars. from previous design versions I guess..

**Review by vladlos — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh and @nathanstilwell)

___
*[pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl, line 49 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNwfOtCBU1RY6uga-5a:-MNxNBQi53t1WaTtRznj:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/397aef9225765a2ad07f378ef79914ae3d36129e/pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl#L49)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

This doesn't fill like a consistent solution, instead of adjusting class names, I would suggest to refactor places where `nodes-table__link` class is used. Particularly, define this class exactly on `<Link>` or `<a>` elements

For instance, one of code blocks can be changed from this:
```jsx
<div className="cell--show-on-hover nodes-table__link">
    <Link to={`/node/${record.nodeId}/logs`}>Logs</Link>
</div>
```

to this:
```jsx
<div className="cell--show-on-hover">
    <Link to={`/node/${record.nodeId}/logs`} className="nodes-table__link">Logs</Link>
</div>
```
</blockquote></details>

Done.

**Review by koorosh — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r4.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @elkmaster, @koorosh, and @nathanstilwell)

___
*[pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl, line 49 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNwfOtCBU1RY6uga-5a:-MO1CUHHAK9aaUpvJSA1:b-v3x2ks) ([raw file](https://github.com/cockroachdb/cockroach/blob/397aef9225765a2ad07f378ef79914ae3d36129e/pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl#L49)):*
<details><summary><i>Previously, elkmaster wrote…</i></summary><blockquote>

Done.
</blockquote></details>

One more place where this class name is used: Statement Details page > Diagnostics tab
"All statement diagnostics" link still has old colors

**Review by vladlos — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh and @nathanstilwell)

___
*[pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl, line 49 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/57130#-MNwfOtCBU1RY6uga-5a:-MO1l-74Bh1UtPDtB-St:bb7f2mf) ([raw file](https://github.com/cockroachdb/cockroach/blob/397aef9225765a2ad07f378ef79914ae3d36129e/pkg/ui/src/views/cluster/containers/nodesOverview/nodes.styl#L49)):*
<details><summary><i>Previously, koorosh (Andrii Vorobiov) wrote…</i></summary><blockquote>

One more place where this class name is used: Statement Details page > Diagnostics tab
"All statement diagnostics" link still has old colors



</blockquote></details>

done

**Review by koorosh — APPROVED** [review]
> Reviewed 6 of 6 files at r5.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57130)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh and @nathanstilwell)

**Pull request merged**